apps/odyssey-stats/src/styles/typography.scss

```diff
@@ -9,6 +9,14 @@
 	@include stats-section-header-for-jetpack;
 }
 
+// Replace fonts in client/my-sites/stats/stats-module-utm-builder/style.scss.
+// Need to be pretty specific to take precedence over those rules.
+.is-odyssey-stats.stats-utm-builder__overlay
```

Comment:
We might not need to introduce an Odyssey Stat dedicated class name `.is-odyssey-stats` to override CSS styles. For example: `.wp-admin .stats-utm-builder__overlay`.

Reply:
I've opened [a new Issue](https://github.com/Automattic/red-team/issues/352) for this on the maintenance board. Thanks for the clarification, @dognose24.